specs/experimental/l2_blob_tx.md

```diff
@@ -0,0 +1,43 @@
+# L2 Blob Transaction
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [L2 Blob Transaction](#l2-blob-transaction)
+  - [Motivation](#motivation)
+  - [How It Works](#how-it-works)
+  - [Reference Implementation](#reference-implementation)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+To make it seamless for dapps that's depending on blob transaction to migrate from L1 to L2, it's important that L2 also supports blob transaction. This will also make L3 more uniform since they can also use blob transaction for batch submitting, just like L2.
```

Comment:
The objective of the proposal is to add BLOB tx support to OP Stack. Describe blob-based txs (examples) that are becoming popular on L1. But L1 is expensive, so it is important to support them in OP Stack to minimize migration cost.

Reply:
By default, all BLOB and calldata will use L1 as DA, but we may further explain the proposal can enable hybrid (forget the exact) DA so that
- High-value applications can use L2 calldata and L1 as DA;
- Low-value applications can use L2 BLOB and DAC/Plasma as DA.